src/software/backend/input/network/networking/network_client.cpp

```diff
@@ -13,11 +13,13 @@ NetworkClient::NetworkClient(std::string vision_multicast_address,
                              int gamecontroller_multicast_port,
                              std::function<void(World)> received_world_callback)
     : network_filter(),
+      world(),
       io_service(),
       last_valid_t_capture(std::numeric_limits<double>::max()),
       initial_packet_count(0),
       received_world_callback(received_world_callback)
 {
+    world.updateRefboxGameState(RefboxGameState::HALT);
```

Comment:
This is key

Reply:
This should not have been committed. If it's required, then I don't think this is the right fix. Removing it and checking now.

Reply:
Doesn't look like this is needed, could you please verify that just the modifications to `game_state.cpp` are not sufficient?

Reply:
hm it works now. Maybe I modified the game state constructor differently